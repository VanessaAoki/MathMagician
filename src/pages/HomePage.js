/* eslint-disable max-len */
import React from 'react';

const HomePage = () => (
  <div className="homeContainer">
    <h2>Welcome to our page!</h2>
    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Quia itaque harum, corporis atque est modi totam similique asperiores. Repellat, aut?</p>
    <p>Cupiditate minima fugiat consequuntur dolor porro dolorem non! Asperiores officia nam delectus, temporibus enim nemo deleniti quisquam explicabo unde sit.</p>
  </div>
);

export default HomePage;
