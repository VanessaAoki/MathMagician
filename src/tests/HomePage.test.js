import React from 'react';
import renderer from 'react-test-renderer';
import HomePage from '../pages/HomePage';

it('Home renders as expected', () => {
  const tree = renderer.create(<HomePage />).toJSON();
  expect(tree).toMatchSnapshot();
});
