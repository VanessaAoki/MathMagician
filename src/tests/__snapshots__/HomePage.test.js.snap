// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders as expected 1`] = `
<h2>
  Home
</h2>
`;
